### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by Xendergo** [Xendergo/fabric-remote-monitor#12]
I was thinking LGPL, but I would like input from other people, especially from literally the only other person who knows this project exists (matgenius04)

**Comment by Matgenius04** [comment]
> I was thinking LGPL, but I would like input from other people, especially from literally the only other person who knows this project exists (matgenius04)

Hmm. I think just normal GPL might be better just to keep everything more open source. We can also always change the license later, right?

**Comment by Xendergo** [comment]
Yeah, I changed my mind later bc like what's the point, this isn't a library anyways

**Comment by Xendergo** [comment]
https://github.com/Xendergo/fabric-remote-monitor/commit/39469fa665888fdb5101a13473018e0cff4605e9

**Comment by Xendergo** [comment]
Actually, I just found this: https://www.gnu.org/licenses/why-affero-gpl.html

I think that might be kinda important in our circumstance
